content/devtools/overview.md

```diff
@@ -9,7 +9,7 @@ async function bootstrap() {
   const app = await NestFactory.create(AppModule, {
     snapshot: true,
   });
-  await app.listen(3000);
+  await app.listen(8001);
```

Comment:
Updated the relevant image to show port 8001


https://null/null/blob/2fa2a98c1c8fa7a3387d5b319b4754bf45bbd5a6/content/devtools/overview.md#L89

Reply:
The application can run on any port you wish (3000 is fine). `DevtoolsHttpModule` exposes another HTTP server which by default uses 8000 (so there're basically 2 HTTP servers)